content/quickstart/aws/install-language-runtime.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Install Language Runtime
+weight: 3
+menu:
+  quickstart:
+    parent: aws
+    identifier: aws-install-language-runtime
+---
+
+## Choose Your Language
```

Comment:
Should this use the language switcher? Because it's sticky, doing so would mean the rest of the steps from here onward would automatically be customized to your selection.

Reply:
I originally did this, but our current lang chooser tabs didn’t quite look/feel right above regular text like the tabs do above code snippets. But it’s a good point: if we capture the choice here, then it’ll be primed for the rest of the steps.

Reply:
Fixed

<img width="374" alt="Screen Shot 2019-06-14 at 12 37 10 PM" src="https://user-images.githubusercontent.com/710598/59533694-23063900-8ea1-11e9-927e-6ad6470216e2.png">